languages/nim/exercises/concept/basics/.meta/design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
```

Comment:
I think this list should also include knowing how to define comments (as the stub file will have them)